roles/hxr.admin-tools/defaults/main.yml

```
@@ -15,11 +15,10 @@ admin_packages:
   - net-tools
   - unzip
   - mutt
-  - byobu
+  - "{{ 'byobu' if ansible_distribution_major_version < '9' else default(omit) }}"
```

```suggestion
  - "{{ 'byobu' if ansible_distribution_major_version < '9' else omit }}"
```

Default is a filter, it cannot go there.

```
    - name: Debug.
      ansible.builtin.debug:
        msg: "{{ 'byobu' if ansible_distribution_major_version < '8' else default(omit) }}"
```
```
TASK [Debug.] ************************************************************************************************************************************************************************
fatal: [sn06.galaxyproject.eu]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'default' is undefined\n\nThe error appears to be in '/home/jose/Dokumente/Repositories/usegalaxy-eu_infrastructure-playbook/htcondor.yml': line 5, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n  pre_tasks:\n    - name: Debug.\n      ^ here\n"}
```

---

Is anyone except me using byobu here?

---

tmux user here ...